src/zenml/cli/server.py

```diff
@@ -30,112 +30,112 @@
 from zenml.logger import get_logger
 
 logger = get_logger(__name__)
-SERVICE_CONFIG_FILENAME = "ZenService.json"
-GLOBAL_ZENML_SERVICE_CONFIG_FILEPATH = os.path.join(
-    get_global_config_directory(), SERVICE_CONFIG_FILENAME
+SERVER_CONFIG_FILENAME = "ZenServer.json"
+GLOBAL_ZENML_SERVER_CONFIG_FILEPATH = os.path.join(
+    get_global_config_directory(), SERVER_CONFIG_FILENAME
 )
 
 
 @cli.group(
     cls=TagGroup,
     tag=CliCategories.MANAGEMENT_TOOLS,
 )
-def service() -> None:
+def server() -> None:
     """ZenML server."""
```

Comment:
Should this be ZenServer?

Reply:
I guess this question applies to any other place where we talk about 'ZenML server'.